packages/chaire-lib-backend/src/utils/processManagers/osrmProfiles/rail.lua

```diff
@@ -0,0 +1,138 @@
+-- Copyright 2017-2019 Railnova SA <support@railnova.eu>, Nikita Marchant <nikita.marchant@gmail.com>
+-- Code under the 2-clause BSD license
+
+api_version = 4
```

Comment:
Where does this file come from? Is it copy-pasted from anywhere? If from osrm, then why do we need it in Transition and don't use the default from there? If it is modified from a copy-paster of osrm, could you add a comment at the top of the file explaining why you need to modify the default rail profile (so that we know right away when opening the file later) and also a comment in the file itself where the modification was done (to clearly highlight the change)

Reply:
osrm does not include any rail profile, ans this one was wroking well. Including the copyright with license is not enough?

Reply:
This copyright does not tell where it comes from. This code was not produced for Transition, these people published it somewhere. And when we need to update it 2 years from now, we want to be able to track it.

Reply:
added source + commit msg